doczrc.js

```diff
@@ -26,6 +26,9 @@ export default {
         p, a, h1, h2, h3, h4, table {
           line-height: 1.6;
         }
+        .bXQPyO, .iCcihC {
```

Comment:
I'm not sure if it's too fragile to target SC classes, but I'm not sure if Docz has a setting for overriding logo styles 🤔

Reply:
Yes, I think that's a bit fragile.

Maybe we can create a custom `Page` component and a `Sidbar` component to change the styling. Not sure how to do this but the following [migration guide](https://github.com/pedronauck/docz/blob/master/MIGRATION_GUIDE.md#creating-and-using-docz-themes) looks interesting.

But I think we can add a comment to that style and improve this later.